ignite/pkg/cosmosgen/generate_go.go

```diff
@@ -6,22 +6,13 @@ import (
 
 	"github.com/otiai10/copy"
 	"github.com/pkg/errors"
-
-	"github.com/ignite/cli/ignite/pkg/protoanalysis"
-	"github.com/ignite/cli/ignite/pkg/protoc"
 )
 
-var goOuts = []string{
-	"--gocosmos_out=plugins=interfacetype+grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:.",
-	"--grpc-gateway_out=logtostderr=true:.",
+func (g *generator) goTemplate() string {
```

Comment:
Maybe to keep consistency with `openAPITemplate` and `tsTemplate`

```suggestion
func (g *generator) gogoTemplate() string {
```